### PR TITLE
Minor improvements to grammar, verbiage and others

### PR DESCRIPTION
**Pull request by buunguyen**
The README and CONTRIBUTING have been super useful for a Babel newbie like me. Thought I could improve a few things to make them better.

### CODE REVIEW

**Comment by hzoo** [comment]
cool thanks 👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,20 +27,20 @@ Contributions are always welcome, no matter how large or small.
 
 - If you aren't just making a documentation change, you'll probably want to learn a bit about a few topics.
  - [ASTs](https://en.wikipedia.org/wiki/Abstract_syntax_tree) (Abstract Syntax Tree): The Babel AST [spec](https://github.com/babel/babylon/blob/master/ast/spec.md) is a bit different from [ESTree](https://github.com/estree/estree). The differences are listed [here](https://github.com/babel/babylon#output).
- - This repository's [`/doc`](https://github.com/babel/babel/tree/master/doc) directory for notes on Babel's internals
+ - This repository [`/doc`](https://github.com/babel/babel/tree/master/doc) contains the notes on Babel's internals
```
Comment (existentialism) [inline CONTRIBUTING.md]: > This repository `/doc` contains the notes on Babel's internals

Reads weird to me?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,20 +27,20 @@ Contributions are always welcome, no matter how large or small.
...
  - Check out [the Babel Plugin Handbook](https://github.com/thejameskyle/babel-handbook/blob/master/translations/en/plugin-handbook.md#babel-plugin-handbook) - core plugins are written the same way as any other plugin!
  - Check out [AST Explorer](http://astexplorer.net/#/scUfOmVOG5) to learn more about ASTs or make your own plugin in the browser
-- When you feel ready to finally jump into the babel source code a good start is to look out for issues which are labeled with [help-wanted](https://github.com/babel/babel/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and/or [beginner-friendly](https://github.com/babel/babel/issues?q=is%3Aissue+is%3Aopen+label%3A%22beginner-friendly%22).
+- When you feel ready to finally jump into the Babel source code, a good start is to look out for issues which are labeled with [help-wanted](https://github.com/babel/babel/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and/or [beginner-friendly](https://github.com/babel/babel/issues?q=is%3Aissue+is%3Aopen+label%3A%22beginner-friendly%22).
```
Comment (existentialism) [inline CONTRIBUTING.md]: Maybe simpler as:

> When you feel ready to jump into the Babel source code, a good place to start is to look for issues tagged with [help-wanted] and/or [beginner-friendly].

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -27,20 +27,20 @@ Contributions are always welcome, no matter how large or small.
...
 
-Feel free to check out the `#discussion`/`#development` channels on our [slack](https://slack.babeljs.io). Some of us are always online to chat!
+Feel free to check out the `#discussion`/`#development` channels on our [Slack](https://slack.babeljs.io). Some of us are always online to chat!
 
 ## Developing
 
 **Note:** Versions `< 5.1.10` can't be built.
 
-Babel is built for node 4 and up but we develop using node 6. Make sure you are on npm 3.
+Babel is built for Node 4 and up but we develop using Node 6. Make sure you are on NPM 3.
```
Comment (existentialism) [inline CONTRIBUTING.md]: Nit: but npm should be in all lower, and maybe do Node.js?

File: README.md [hunk 4]
```diff
@@ -128,19 +128,19 @@ Check out the [`babel-handbook`](https://github.com/thejameskyle/babel-handbook/
...
-- [`babel-template`](/packages/babel-template) is a helper function to make AST nodes. Instead you can pass a string representing the code you want to create rather than tediously building them using `babel-types`.
-- [`babel-helpers`](/packages/babel-helpers) is a set of premade `babel-template` functions that are used in some babel plugins.
-- [`babel-code-frame`](/packages/babel-code-frame) is a standalone package used to generate errors that prints the source code and points to error locations.
+- [`babel-register`](/packages/babel-register) is a way to automatically compile files with babel on the fly by binding to NodeJS's `require`. Check out the [docs](http://babeljs.io/docs/usage/require/).
+- [`babel-template`](/packages/babel-template) is a helper function to create AST nodes by passing a string representing the code instead of tediously building them using `babel-types`.
+- [`babel-helpers`](/packages/babel-helpers) is a set of pre-made `babel-template` functions that are used in some babel plugins.
```
Comment (existentialism) [inline README.md]: babel -> Babel

File: README.md [hunk 5]
```diff
@@ -128,19 +128,19 @@ Check out the [`babel-handbook`](https://github.com/thejameskyle/babel-handbook/
...
-- [`babel-register`](/packages/babel-register) is a way to automatically compile files with babel on the fly by binding to node's require. Check out the [docs](http://babeljs.io/docs/usage/require/).
-- [`babel-template`](/packages/babel-template) is a helper function to make AST nodes. Instead you can pass a string representing the code you want to create rather than tediously building them using `babel-types`.
-- [`babel-helpers`](/packages/babel-helpers) is a set of premade `babel-template` functions that are used in some babel plugins.
-- [`babel-code-frame`](/packages/babel-code-frame) is a standalone package used to generate errors that prints the source code and points to error locations.
+- [`babel-register`](/packages/babel-register) is a way to automatically compile files with babel on the fly by binding to NodeJS's `require`. Check out the [docs](http://babeljs.io/docs/usage/require/).
```
Comment (existentialism) [inline README.md]: Nit: babel -> Babel, NodeJS -> Node.js

File: README.md [hunk 6]
```diff
@@ -128,19 +128,19 @@ Check out the [`babel-handbook`](https://github.com/thejameskyle/babel-handbook/
...
+- [`babel-template`](/packages/babel-template) is a helper function to create AST nodes by passing a string representing the code instead of tediously building them using `babel-types`.
```
Comment (existentialism) [inline README.md]: > is a helper function to create AST nodes by

Reads weird, IMO.

Quick stab, feel free to edit:

> is a helper function that eliminates the tedium of creating AST nodes through `babel-types` by allowing you to pass a string representation of the code instead.

File: CONTRIBUTING.md (hunk 3 above)
Comment (buunguyen) [inline CONTRIBUTING.md]: Great, I didn't know that. Always thought it should be NPM :).

File: README.md (hunk 6 above)
Comment (buunguyen) [inline README.md]: Another attempt https://github.com/babel/babel/pull/5766/files#diff-04c6e90faac2675aa89e2176d2eec7d8R135 :)

**Pull request merged**